app.go

```diff
@@ -72,19 +72,27 @@ type (
 	}
 )
 
-var prefork = flag.Bool("fiber-prefork", false, "use prefork")
-var child = flag.Bool("fiber-child", false, "is child process")
+var prefork bool
+var child bool
 
-// New: https://fiber.wiki/application#new
+func init() {
```

Comment:
but `init()` is bad practice for testing, as I get comment on Dev.to 🤔

Reply:
My gut feeling is that it was better with `fiber` prefix to avoid collision with end-user flags. Or maybe no flags at all and pass everything via `Settings`?

Reply:
We need the flags because we are working with separate child processes using prefork. They do not share memory. I solved the `flag.Parse()` conflict looking at this solution: https://github.com/valyala/fasthttp/blob/master/prefork/prefork.go#L45,L60